src/pretix/static/datetimepicker/bootstrap-datetimepicker.js

```diff
@@ -947,7 +947,7 @@
 
                 input.blur();
 
-                viewDate = date.clone();
+                //viewDate = date.clone();
```

Comment:
The PR body says that the dependency does not need to be modified, so is this change unintentionally kept?

Reply:
Yup, sorry, that is not intentional.